container-messagebus/src/main/java/com/yahoo/messagebus/jdisc/test/RemoteClient.java

```diff
@@ -40,28 +46,22 @@ public Reply awaitReply(int timeout, TimeUnit unit) throws InterruptedException
         return queue.awaitReply(timeout, unit);
     }
 
-    public String slobrokId() {
-        return slobrokId;
+    public SlobroksConfig slobroksConfig() {
+        return TestUtils.configFor(slobrok);
```

Comment:
Should this method be used "network==true" (production/non-test?) code path in constructor?

Reply:
Did not notice the package name...